### ISSUE TITLE
When Pretraining on CoCo, the acc is low

### ISSUE THREAD

**Issue by HaloTrouvaille** [facebookresearch/SLIP#7]
When pre training on COCO, after 9 epochs training, the zero-shot acc of ViTB is still 0.100.
I follow the instruction, --lr 3e-3 --wd 0.1. 
Since I have 2 GPUs, I set batchsize=8, and --update-fre= 256 to keep total  batchsize 4096.

**Comment by normster** [comment]
Hi Halo,

COCO is a much smaller dataset than YFCC15M so we would expect to see much lower performance especially with only 9 epochs of training. However, 0.100 is exactly chance performance on ImageNet which suggests an issue with the optimization itself, specifically a batch size of 8 is likely causing issues with the BatchNorm layers in the SimCLR mlp. I haven't tried this myself but you can try replacing it with GroupNorm + Weight Standardization (per [BiT](https://arxiv.org/abs/1912.11370)) or LayerNorm (per [CPC v2](https://arxiv.org/abs/1905.09272)), or just removing Batch Norm entirely (per [MoCo v3](https://arxiv.org/abs/2104.02057)). You can also try training CLIP instead, which doesn't use Batch Norm.

**Comment by HaloTrouvaille** [comment]
Thank you for your suggestion, I will try.
